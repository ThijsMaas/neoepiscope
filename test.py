# from collections import defaultdict

# l = [('chr19', 17807975, 'AG', '', 'D', 0.132, 'NA', 'NA', 0, 'ENST00000318683.7'),
#     ('chr19', 17807981, 'GG', '', 'D', 0.105, 'NA', 'NA', 0, 'ENST00000318683.7'),
#     ('chr19', 17807975, 'AG', '', 'D', 0.132, 'NA', 'NA', 0, 'ENST00000595387.1'),
#     ('chr19', 17807981, 'GG', '', 'D', 0.105, 'NA', 'NA', 0, 'ENST00000595387.1')]

# for m in set(mutation[0:7] for mutation in l):
#     print(m)

# mutation_dict = defaultdict(list)
# for mut in l:
#     mutation_dict[(mut[0:7])+(mut[8],)].append([mut[7], mut[9]])
# print(mutation_dict)
