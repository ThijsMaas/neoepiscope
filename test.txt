

bam /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/test/update_output/recal/tumor_neogene_1.bam



extractHAIRS --indels 1 \
    --bam /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/test/update_output/recal/tumor_neogene_1.bam \
    --VCF /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/test/update_output/calls/merged_neogene_1_split.vcf \
    --out /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/test/update_output/calls/fragments_neogene_1_split \
           && \
       HAPCUT2  \
           --fragments /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/test/update_output/calls/fragments_neogene_1_split \
           --VCF /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/test/update_output/calls/merged_neogene_1_split.vcf \
           --output /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/test/update_output/calls/adjusted_haplotypes_neogene_1_split

neoepiscope prep \
            -v /home/t.maas/neogene/neogene-snv-docker/test/update_output/calls/merged_neogene_1.vcf \
            -o test_haplotypes


neoepiscope call     \
-x /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/reference/hg38/GRCh38.p13.genome.fa     \
-d /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/reference/hg38/     \
-c /home/t.maas/neogene/neogene-snv-docker/test/update_output/epitopes/adjusted_haplotypes_neogene_1     \
-v /home/t.maas/neogene/neogene-snv-docker/test/update_output/calls/somatic_neogene_1_filtered_split.vcf     \
-o test_neo0     \
--no-affinity     \
--kmer-size 25

neoepiscope call     \
-x /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/reference/hg38/GRCh38.p13.genome.fa     \
-d /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/reference/hg38/     \
-c test/adjusted_haplotype     \
-v test/test.vcf     \
-o test_neo19     \
--no-affinity     \
--kmer-size 25

neoepiscope call     \
-x /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/reference/hg38/GRCh38.p13.genome.fa     \
-d /data/bambino/scratch/t.maas/neogene/neogene-snv-docker/reference/hg38/     \
-c test/examples_haplotype     \
-v test/examples.vcf     \
-o test/examples_epitopes.tsv     \
--no-affinity     \
--kmer-size 25

