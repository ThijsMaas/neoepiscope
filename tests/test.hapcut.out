BLOCK: offset: ...
1	1	0	11	71276862	TGT	T	0/0:0%	0	0.0	100.0
2	1	0	11	71276900	C	G	0/0:0%:	0	0.0	100.0
********
BLOCK: offset: ...
4	1	1	11	71279000	TGT	T	0/0:0%	0	0.0	100.0
5	1	0	11	71279100	C	G	0/0:0%	0	0.0	100.0
